packages/calendar/Calendar.tsx

```diff
@@ -0,0 +1,211 @@
+import { h, Component, prop, emit } from 'skatejs';
+import styles from './Calendar.scss';
+
+import * as getYear from 'date-fns/get_year';
+import * as getMonth from 'date-fns/get_month';
+import * as addDays from 'date-fns/add_days';
+import * as getDate from 'date-fns/get_date';
+import * as startOfWeek from 'date-fns/start_of_week';
+import * as format from 'date-fns/format';
+import * as isToday from 'date-fns/is_today';
+import * as parse from 'date-fns/parse';
+import * as isEqual from 'date-fns/is_equal';
+import { css } from '../_helpers/css';
+import { Button } from '../button/Button';
+import { buildFormatLocale, LocaleType } from '../_helpers/buildFormatLocale';
+
+interface CalendarProps extends JSX.HTMLProps<HTMLElement | any> {
+  selectedDate?: string,
+  onDateChange?: Function,
+  i18n?: {
+    monthsFull: string[],
+    weekdays2char: string[],
+  },
+  weekStartsOn?: number,
+}
+
+export class Calendar extends Component<CalendarProps> {
+
+  constructor() {
+    super();
+
+    this.prevYear = this.prevYear.bind( this );
+    this.nextYear = this.nextYear.bind( this );
+    this.prevMonth = this.prevMonth.bind( this );
+    this.nextMonth = this.nextMonth.bind( this );
+    this.setDate = this.setDate.bind( this );
+  }
+
+  static get is() { return 'bl-calendar'; }
+
+  static get props() {
+    return {
+      days: prop.array(),
```

Comment:
We need to call renderCallback() when days or (years|months) are changed, so this is reason what is here.